src/docker.c

```diff
@@ -33,7 +33,7 @@
 #include <curl/curl.h>
 
 // Default size of the response buffer when calling the Docker stats API
-#define RESPONSE_BUFFER_SIZE 128000
+#define RESPONSE_BUFFER_SIZE 1024000
 
```

Comment:
Unrelated to this PR, right? Can we pull this change out into a separate PR?

Reply:
We can remove this, as @dhrupadb made a change in his branch to reflect the same thing. This was just committed so I could produce a build out of it.